Knockback resistance & Attack knockback (#186)
MCreator/MCreator: plugins/mcreator-localization/help/default/entity/attack_knockback.md

The attack knockback defines the knockback value of each attack of this entity

> In general, most of Vanilla entities don't use it, but as a reference, the Ravager has 1.5.

Reviewer: ```suggestion
Most vanilla entities do not use it. Some do, such as the Ravager, which has a knockback value of 1.5.
```
split it. makes it easier to read.

Reviewer 2: Also vanilla should be lowercase, same with ravager
